allure-web/src/components/execution/steps-list.hbs

```diff
@@ -1,26 +1,31 @@
 {{#each steps}}
     <div class="step">
-        <div class="{{b "step" "title" hasContent=(or steps.length attachments.length)}}">
-            {{#if (or steps.length attachments.length)}}
+        <div class="{{b "step" "title" hasContent=(or steps.length attachments.length parameters.length)}}">
+            {{#if (or steps.length attachments.length parameters.length)}}
                 <span class="step__arrow block__arrow">{{arrow status}}</span>
             {{else}}
                 <span class="step__status">{{allure-icon status}}</span>
             {{/if}}
+            <div class="step__name">
             {{name}}
-            {{#if parameters}}
-                [
-                {{#each parameters}}
-                    <span>{{name}}: {{value}}</span>
-                {{/each}}
-                ]
-            {{/if}}
+            </div>
```

Comment:
Have you decided not to display params at step title at all?

I probably would make people who have one short explicit parameter happy.
```
Login user [test-username]
```

For people who has more than one parameter, or single parameter, but freaky long, it would become the following

```
Login user [x parameters]
```